.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "riscv-tests"]
+	path = programs/tests/riscv
+	url = git@github.com:riscv-software-src/riscv-tests.git
```

Comment:
If this is a public repo it should use the https link (otherwise when cloning on a keyless machine it may give errors)